x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/runtimefields/RuntimeFieldsFeatureSetUsage.java

```diff
@@ -217,6 +244,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field("count", count);
             builder.field("index_count", indexCount);
             builder.field("scriptless_count", scriptLessCount);
+            builder.field("shadowed_count", shadowedCount);
```

Comment:
is it shadowed, or is it shadowing, or? :)

Reply:
Well the runtime fields is doing the shadowing, but we're also counting the number of fields shadowed by the runtime section, so :shrug: - I can change to `shadowing_count` if you think that's clearer.  Or maybe have another name altogether...

Reply:
it's fine as-is thanks